### ISSUE TITLE
當使用手機裝置時會無法滾動（ain't scrolling when use mobile device）

### ISSUE THREAD

**Issue by dolphinlin** [iissnan/hexo-theme-next#1639]
### Expected behavior (预期行为)
滾動頁面（手機裝置）
Scrolling page (Mobile device)

mobile device: iphone 6
whether i use Chrome or Safari, it all comes to the same thing

### Actual behavior (实际行为)
無法滾動
ain't scrolling

### Steps to reproduce the behavior (重现步骤)
設定 緞帶 > canvas_ribbon: true
to set > canvas_ribbon: true

### NexT Information

Add the  ✔  sign before an item which is affected by this behavior.

NexT Version:

  -  ✔Master
  -  Latest Release
  -  Old version - 

NexT Scheme:
  -  All schemes
  -  Muse
  -  Mist
  -  ✔Pisces

**Comment by leviding** [comment]
Me too. iPad mini2.

**Comment by ivan-nginx** [comment]
Need to find author of this feature and ask him for it bug, i think.

**Comment by dolphinlin** [comment]
Who is the author of plugin?
The bug is so serious😫

**Comment by ivan-nginx** [comment]
Hmmm) U can find it by History versions of file, whom give this feature.

**Comment by dolphinlin** [comment]
WTF....i find it!!!!!!
the code cause an exception..
```js

document.addEventListener("touchmove", function(e) {
    e.preventDefault()
  });

``` 

the author say '重写了vue作者博客中的背景彩带' <<
but i go to the site of Evanyou, i found that his page is no scroll bar ....

**Comment by ivan-nginx** [comment]
Ok, i don't uderstand what author say, but u can do pull request with fix on it.

**Comment by dolphinlin** [comment]
he just say that 'rewrite the background ribbon of Vuejs's author website for the plugin'

**Comment by ivan-nginx** [comment]
Mb he's mean, what need to copy last version of any scripts from original author page to fix it?
Anyway, if this error appear on author page, need to contact with this author and say it to him. Mb, create issue on his Github, for example.
***
P.S. Why u need this beauty things? Most of the all — is simple. If anything do wall afead you eyes, that's mostly mean what in context and idea is nothing. This is my opinion.

**Comment by dolphinlin** [comment]
okay! i think simple is beauty tooooo!
i just set the function for fun , LOL
i will pull req for the code

**Comment by ivan-nginx** [comment]
> i will pull req for the code

Ok.

**Comment by ivan-nginx** [comment]
Fix in #1640.

If issue resolved, close it please.
